apigentools/cli.py

```diff
@@ -64,6 +70,12 @@ def get_cli_parser():
         default=env_or_val("APIGENTOOLS_SPEC_DIR", constants.DEFAULT_SPEC_DIR),
         help="Path to directory with OpenAPI specs (default: '{}')".format(constants.DEFAULT_SPEC_DIR),
     )
+    generate_parser.add_argument(
+        '--clone-repo',
+        action='store_true',
+        default=env_or_val('APIGENTOOLS_SKIP_PULL_REPO', False),
```

Comment:
You should use `_type=bool` as argument for `env_or_val`, see documentation of this function.